mungegithub/mungers/submit-queue.go

```diff
@@ -161,10 +162,12 @@ type SubmitQueue struct {
 
 	Committers             string
 	RequiredStatusContexts []string
-	doNotMergeMilestones   []string
+	DoNotMergeMilestones   []string
```

Comment:
no other comments from me.

On Thu, Aug 11, 2016 at 3:59 PM, Anirudh Ramanathan <
notifications@github.com> wrote:

> In mungegithub/mungers/submit-queue.go
> <https://github.com/kubernetes/contrib/pull/1495#discussion_r74519100>:
>
> > @@ -161,10 +162,12 @@ type SubmitQueue struct {
> >
> >  	Committers             string
> >  	RequiredStatusContexts []string
> > -	doNotMergeMilestones   []string
> > +	DoNotMergeMilestones   []string
>
> That makes sense. I'll do that in a follow-up PR.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/contrib/pull/1495/files/8714e01db9d88df7c880f2eb915dae2e818ed545#r74519100>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAnglvFxPnG4oypd5QzvgeFQesVcNqNUks5qe6k6gaJpZM4Jebml>
> .
>